### PR TITLE
fix: remove atomic instruction

### PR DESCRIPTION
**Pull request by XuJiandong**
It can be done without hardfork: https://github.com/nervosnetwork/ckb-std/tree/master?tab=readme-ov-file#upgrading-issues

Additionally, refine the wording related to the syscall spawn.

### CODE REVIEW

File: rfcs/0049-ckb-vm-version-2/0049-ckb-vm-version-2.md [hunk 1]
```diff
@@ -28,11 +27,10 @@ To address the aforementioned issues, we have implemented the following optimiza
...
-4. Refactor ckb-vm to make it thread-safe. Thread-safe ckb-vm can take advantage of modern multi-core CPUs and execute multiple threads in parallel, potentially improving performance and throughput. At the same time, many modern applications and frameworks rely on multi-threading, and a thread-unsafe virtual machine may not be compatible with these technologies.
-5. [Macro-Operation Fusion](https://en.wikichip.org/wiki/macro-operation_fusion). There are 5 MOPs added in VM version 2, there are:
+1. One notable addition is the inclusion of a new system call called "Spawn," In essence, Spawn serves as an alternative to dynamic library calls and Exec.
+2. In response to the growing prevalence of the aarch64 architecture, which is the 64-bit version of the ARM architecture utilized in various devices such as smartphones, tablets, and servers, we have incorporated assembly mode implementation into ckb-vm specifically for aarch64. By running ckb-vm on assembly mode, users can potentially experience improved performance with faster execution times, ultimately enhancing the overall efficiency of the system.
```
Comment (xxuejie) [inline rfcs/0049-ckb-vm-version-2/0049-ckb-vm-version-2.md:31]: I don't think assembly implementation of CKB-VM for aarch64 is tied to version 2.

File: rfcs/0049-ckb-vm-version-2/0049-ckb-vm-version-2.md [hunk 2]
```diff
@@ -28,11 +27,10 @@ To address the aforementioned issues, we have implemented the following optimiza
...
+3. Refactor ckb-vm to make it thread-safe. Thread-safe ckb-vm can take advantage of modern multi-core CPUs and execute multiple threads in parallel, potentially improving performance and throughput. At the same time, many modern applications and frameworks rely on multi-threading, and a thread-unsafe virtual machine may not be compatible with these technologies.
```
Comment (xxuejie) [inline rfcs/0049-ckb-vm-version-2/0049-ckb-vm-version-2.md:32]: Personally I wasn't sure what this refers to

File: rfcs/0049-ckb-vm-version-2/0049-ckb-vm-version-2.md (hunk 1 above)
Comment (XuJiandong) [reply rfcs/0049-ckb-vm-version-2/0049-ckb-vm-version-2.md:31]: We will submit another PR for these 2 issues.

**Comment by mohanson** [comment]
I submitted a new PR to replace this one

https://github.com/nervosnetwork/rfcs/pull/444

**Comment by XuJiandong** [comment]
included in: https://github.com/nervosnetwork/rfcs/pull/444

**Pull request closed without merging**